src/Rokt-Kit.js

```diff
@@ -47,7 +46,7 @@ var constructor = function () {
                 ) {
                     window.Rokt.createLauncher({
                         accountId: accountId,
-                        sandbox: sandboxMode,
+                        sandbox: mParticle.getEnvironment() === 'development',
```

Comment:
I would actually do `window.mParticle`.  Just in case the context ever changes for some reason (it shouldn't but this is just being safe and we do this in other kits.  Just like we call `window.Rokt` above) 
```suggestion
                        sandbox: window.mParticle.getEnvironment() === 'development',
```